runtime/doc/message.txt

```diff
@@ -671,15 +671,6 @@ This mostly happens when the disk is full.  Vim could not write text into the
 text may be lost without recovery being possible.  Vim might run out of memory
 when this problem persists.
 
-						*connection-refused*  >
-  Xlib: connection to "<machine-name:0.0" refused by server
-
-This happens when Vim tries to connect to the X server, but the X server does
-not allow a connection.  The connection to the X server is needed to be able
-to restore the title and for the xterm clipboard support.  Unfortunately this
-error message cannot be avoided, except by disabling the |+xterm_clipboard|
-and |+X11| features.
-
```

Comment:
Actually, I'm not so sure. The error message people would get in nvim should be localized to the clipboard, right? Given that, I don't think the above snippet was of much use, especially since it makes it seem like Nvim is communicating via Xlib and that Nvim can "restore the title".

I'm already planning on cleaning up the X11 related docs and plan on merging clipboard information from there into `options.txt:'clipboard'` and/or `nvim_clipboard.txt`, so I'd rather not expand the scope of this PR for this.